spec/classes/server/ldap_spec.rb

```diff
@@ -5,7 +5,7 @@
     on_supported_os.each do |os, facts|
       context "on #{os}" do
         let(:facts) do
-          if ['RedHat','CentOS'].include?(facts[:operatingsystem]) && facts[:operatingsystemmajrelease].to_s < '7'
+          if facts[:operatingsystemmajrelease].to_s < '7'
```

Comment:
This really should be matched against the supported list as it was before. It's more tedious, but we have added SuSE to a few modules and may add it to more and we'll have to go through and re-fix all of these.

Reply:
This list just seems like a duplicate effort, knowing how the facts get there.